libcudacxx/test/libcudacxx/std/text/charconv/charconv.to.chars/integral.pass.cpp

```diff
@@ -143,1550 +143,1550 @@ template <>
-    TestItem{14, "e"},
-    TestItem{16, "g"},
-    TestItem{23, "n"},
-    TestItem{32, "w"},
-    TestItem{36, "12"},
-    TestItem{127, "3p"},
-    TestItem{200, "5u"},
-    TestItem{255, "7h"},
-    TestItem{8158, "71w"},
-    TestItem{50243, "19fp"},
-    TestItem{32767, "sbp"},
-    TestItem{87231, "27fl"},
-    TestItem{65535, "1mnh"},
-    TestItem{17875098, "dcqu6"},
-    TestItem{1787987597, "15bx82h"},
-    TestItem{2147483647, "1d8xqrp"},
-    TestItem{4294967295, "2qhxjlh"},
-    TestItem{657687465411, "chp78tkn"},
-    TestItem{89098098408713, "1fubx38kal"},
-    TestItem{4987654351689798, "2e4wnws450i"},
-    TestItem{9223372036854775807, "3tdtk1v8j6tpp"},
-    TestItem{-1, "-1", "7orp63sh4dphh"},
-    TestItem{-13, "-d", "7orp63sh4dph5"},
-    TestItem{-128, "-3q", "7orp63sh4dpdq"},
-    TestItem{-879, "-pt", "7orp63sh4dopn"},
-    TestItem{-12345, "-an3", "7orp63sh4desf"},
-    TestItem{-32768, "-sbq", "7orp63sh4cv5q"},
-    TestItem{-5165781, "-3temp", "7orp63sh0iasr"},
-    TestItem{-97897347, "-258q9p", "7orp63sex4x7r"},
-    TestItem{-2147483648, "-1d8xqrq", "7orp63r3tdwnq"},
-    TestItem{-165789751156, "-35auvdpm", "7orp60n67gbpu"},
-    TestItem{-8798798743521135, "-48v3mdkohmv", "7ong9063hn7sl"},
-    TestItem{-9223372036854775807 - 1, "-3tdtk1v8j6tpq", "3tdtk1v8j6tpq"},
+    TestItem{0ll, "0"},
+    TestItem{1ll, "1"},
+    TestItem{2ll, "2"},
+    TestItem{3ll, "3"},
+    TestItem{4ll, "4"},
+    TestItem{7ll, "7"},
+    TestItem{8ll, "8"},
+    TestItem{14ll, "e"},
+    TestItem{16ll, "g"},
+    TestItem{23ll, "n"},
+    TestItem{32ll, "w"},
+    TestItem{36ll, "12"},
+    TestItem{127ll, "3p"},
+    TestItem{200ll, "5u"},
+    TestItem{255ll, "7h"},
+    TestItem{8158ll, "71w"},
+    TestItem{50243ll, "19fp"},
+    TestItem{32767ll, "sbp"},
+    TestItem{87231ll, "27fl"},
+    TestItem{65535ll, "1mnh"},
+    TestItem{17875098ll, "dcqu6"},
+    TestItem{1787987597ll, "15bx82h"},
+    TestItem{2147483647ll, "1d8xqrp"},
+    TestItem{4294967295ll, "2qhxjlh"},
+    TestItem{657687465411ll, "chp78tkn"},
+    TestItem{89098098408713ll, "1fubx38kal"},
+    TestItem{4987654351689798ll, "2e4wnws450i"},
+    TestItem{9223372036854775807ll, "3tdtk1v8j6tpp"},
+    TestItem{-1ll, "-1", "7orp63sh4dphh"},
+    TestItem{-13ll, "-d", "7orp63sh4dph5"},
+    TestItem{-128ll, "-3q", "7orp63sh4dpdq"},
+    TestItem{-879ll, "-pt", "7orp63sh4dopn"},
+    TestItem{-12345ll, "-an3", "7orp63sh4desf"},
+    TestItem{-32768ll, "-sbq", "7orp63sh4cv5q"},
+    TestItem{-5165781ll, "-3temp", "7orp63sh0iasr"},
+    TestItem{-97897347ll, "-258q9p", "7orp63sex4x7r"},
+    TestItem{-2147483648ll, "-1d8xqrq", "7orp63r3tdwnq"},
+    TestItem{-165789751156ll, "-35auvdpm", "7orp60n67gbpu"},
+    TestItem{-8798798743521135ll, "-48v3mdkohmv", "7ong9063hn7sl"},
+    TestItem{-9223372036854775807ll - 1, "-3tdtk1v8j6tpq", "3tdtk1v8j6tpq"},
   }};
 }
 
 template <>
 __host__ __device__ constexpr cuda::std::array<TestItem, 40> get_test_items<35>()
 {
   return {{
```

Comment:
With the move to functions returning arrays, why do we need the double braces anymore?